### PR TITLE
chore: aries-js-worker CI skeleton

### LINKED ISSUE #1310: aries-js-worker: CI skeleton
part of #1238

### PR DESCRIPTION
**Pull request by llorllale**
closes #1310

### CODE REVIEW

File: test/aries-js-worker/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "aries-js-worker-test",
+  "version": "0.1.0",
```
Comment (troyronda) [inline test/aries-js-worker/package.json]: suggest setting private flag to true.

https://docs.npmjs.com/files/package.json#private

File: Makefile [hunk 2]
```diff
@@ -49,8 +48,15 @@ unit-test-wasm: depend
 	@scripts/check_unit_wasm.sh
 
 .PHONY: bdd-test
-bdd-test: clean generate-test-keys agent-rest-docker sample-webhook-docker
-	@scripts/check_integration.sh
+bdd-test: clean generate-test-keys agent-rest-docker sample-webhook-docker bdd-js bdd-go
```
Comment (troyronda) [inline Makefile]: note: this will require local installation of chrome to run bdd-test target on a developer's system.

Comment (troyronda) [reply Makefile]: https://github.com/hyperledger/aries-framework-go/blob/master/docs/test/bdd_tests.md#run-all-tests

File: Makefile [hunk 3]
```diff
@@ -49,8 +48,15 @@ unit-test-wasm: depend
...
+
+.PHONY: bdd-go
+bdd-go:
```
Comment (troyronda) [inline Makefile]: would make sense to match prefix with the parent target. e.g., bdd-test-go & bdd-test-js

File: Makefile (hunk 2 above)
Comment (llorllale) [reply Makefile]: No - puppeteer downloads headless chrome during installation

File: Makefile (hunk 3 above)
Comment (llorllale) [reply Makefile]: done

File: test/aries-js-worker/package.json (hunk 1 above)
Comment (llorllale) [reply test/aries-js-worker/package.json]: done

**Comment by troyronda** [comment]
Works locally on my Mac.
Used latest LTS node version: v12.16.1

**Review by troyronda — APPROVED** [review]

File: scripts/check_js_integration.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+ROOT=`pwd`
+
+echo "Running aries-js-worker integration tests..."
+echo "----> packing aries-js-worker"
+cd $ROOT/cmd/aries-js-worker
+npm install
```
Comment (rolsonquadras) [inline scripts/check_js_integration.sh]: Do we need to add NPM to the pre-requisites https://github.com/hyperledger/aries-framework-go/blob/master/docs/test/build.md#prerequisites-for-running-tests-and-demos ?

**Review by rolsonquadras — APPROVED** [review]

**Review by fqutishat — APPROVED** [review]

File: scripts/check_js_integration.sh (hunk 4 above)
Comment (llorllale) [reply scripts/check_js_integration.sh]: added a couple of notes to:

* docs/test/bdd_tests.md
* docs/test/build.md

File: test/aries-js-worker/test/test.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+/*
```
Comment (troyronda) [inline test/aries-js-worker/test/test.js]: Do we have open issues for more substantial tests?

Comment (llorllale) [reply test/aries-js-worker/test/test.js]: #1238 has been broken into sub tasks and that (big one) is one of those

**Pull request merged**